packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -239,6 +246,29 @@ export class VaBreadcrumbs {
     });
   }
 
+  /**
+   * Changes the route when uswds and useRoute props are true.
+   * Route will be set to the href value in the breadcrumb.
+   */
+  handleRouteChange(e: MouseEvent, href: string): void {
+    e.preventDefault();
+    window.history.pushState(null, '', href);
+    // create event to notify Router
+    const popStateEvent = new PopStateEvent('popstate');
+    // notify Router
+    window.dispatchEvent(popStateEvent);
```

Comment:
I'm not sure what you mean by "notify Router". Is this specifically for [React Router](https://reactrouter.com/en/main)?

Reply:
@jamigibbs Dispatching the event here isn't required to make the navigation work (with v 5). But from my understanding dispatching the event might more closely mimic what React Router is doing so I included it in case it increases compatibility with other versions.